src/UserFunctions.java

```diff
@@ -0,0 +1,129 @@
+public class UserFunctions {
+
+    // This needs the Puzzle class
+    // private Puzzle puzzle;
+
+    private List<String> display;
+
+    // public UserFunctions(Puzzle p) {
+    //  puzzle = p;
+    //  display = new List<>();
+    //}
+
+    public UserFunctions() {
+        //_puzzle = new Puzzle();
+        display = new List<>();
+    }
+
+    private String exitCommand = "exit";
+    private String foundCommand = "found";
+    private String helpCommand = "help";
+    private String loadCommand = "load";
+    private String newCommand = "new";
+    private String saveCommand = "save";
+    private String showCommand = "show";
+    private String shuffleCommand = "shuffle";
```

Comment:
Would it be better to have these Strings as private static final, or do they need to be modifiable? That way we wouldn't be able to accidentally change their values during runtime. If we do go that route, popular java convention is to have static final variables formatted like `EXIT_COMMAND`.